src/main/charts/bitbucket/Changelog.md

```diff
@@ -0,0 +1,61 @@
+# Change Log
+
+## 0.11.0 
+
+**Release date:** 2021-06-09
+
+![AppVersion: 7.12.1-jdk11](https://img.shields.io/static/v1?label=AppVersion&message=7.12.1-jdk11&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* Dckube 348 warning of absent persistent volume (#169) 
+* DCKUBE-307 Do not print logs when testng helm installation. (#168) 
+* Dckube 308 installation output notes (#157) 
+* DCKUBE-282: Update icons to SVG (#164) 
+* DCKUBE-302 Add Bitbucket SSH service and custom annotations. (#152) 
```

Comment:
Can these all be updated so that they are in the same format - `DCKUBE-000: Message`. This comment applied to all of the `Changelog.md`'s